frontend/packages/ux-editor-v3/src/components/config/Expressions/ExpressionContent/ExpressionPreview/SimpleExpressionPreview.test.tsx

```diff
@@ -5,7 +5,7 @@ import type { ServicesContextProps } from 'app-shared/contexts/ServicesContext';
 import { renderWithMockStore } from '../../../../../testing/mocks';
```

Comment:
```suggestion
import { renderWithMockStore } from '@studio/testing/mocks';
```

Reply:
Same here as in [my previous comment](https://github.com/Altinn/altinn-studio/pull/12797#discussion_r1611766890)